python/dgl/data/wikics.py

```diff
@@ -0,0 +1,157 @@
+"""Wiki-CS Dataset"""
+import itertools
+import os
+import json
+import numpy as np
+from .. import backend as F
+from ..convert import graph
+from .dgl_dataset import DGLBuiltinDataset
+from ..transforms import to_bidirected, reorder_graph
+from .utils import generate_mask_tensor, load_graphs, save_graphs, _get_dgl_url
+
+
+class WikiCSDataset(DGLBuiltinDataset):
+    r"""Wiki-CS is a Wikipedia-based dataset for node classification from `Wiki-CS: A Wikipedia-Based
+    Benchmark for Graph Neural Networks <https://arxiv.org/abs/2007.02901v2>`_
+
+    The dataset consists of nodes corresponding to Computer Science articles, with edges based on
+    hyperlinks and 10 classes representing different branches of the field.
+
+    WikiCS dataset statistics:
+
+    - Nodes: 11,701
+    - Edges: 431,726 (note that the original dataset has 216,123 edges but DGL adds
+      the reverse edges and removes the duplicate edges, hence with a different number)
+    - Number of Classes: 10
+    - Node feature size: 300
+    - Number of different train, validation, stopping splits: 20
+    - Number of test split: 1
+
+    Parameters
+    ----------
+    raw_dir : str
+        Raw file directory to download/contains the input data directory.
+        Default: ~/.dgl/
+    force_reload : bool
+        Whether to reload the dataset.
+        Default: False
+    verbose : bool
+        Whether to print out progress information.
+        Default: False
+    transform : callable, optional
+        A transform that takes in a :class:`~dgl.DGLGraph` object and returns
+        a transformed version. The :class:`~dgl.DGLGraph` object will be
+        transformed before every access.
+
+    Attributes
+    ----------
+    num_classes : int
+        Number of node classes
+
+    Examples
+    --------
+    >>> dataset = WikiCSDataset()
+    >>> dataset.num_classes
+    10
+    >>> g = dataset[0]
+    >>> # get node feature
+    >>> feat = g.ndata['feat']
+    >>> # get node labels
+    >>> labels = g.ndata['label']
+    >>> # get data split
+    >>> train_mask = g.ndata['train_masks']
+    >>> val_mask = g.ndata['val_masks']
+    >>> stopping_mask = g.ndata['stopping_masks']
+    >>> test_mask = g.ndata['test_mask']
```

Comment:
Describe the shape of the mask tensors. For tensors of shape (*, 20), explain what `20` stands for.